cookbooks/imos_webapps/files/default/geonetwork/geonetwork-config-manager.rb

```
@@ -2,8 +2,7 @@
 
```

Script fails silently when supplied with the wrong password. 

`sudo /opt/chef/embedded/bin/ruby geonetwork-config-manager.rb -g http://172.28.128.3:8081/geonetwork -c geonetwork-config-geonetwork.json -u admin -p admin1111
`